### ISSUE TITLE
Wait for start/stop/etc.

### ISSUE THREAD

**Issue by fizruk** [fizruk/lxc#5]
When performing actions with a container one may want to wait after each `start`, `stop`, `freeze`, `unfreeze` commands. The code would look like:

``` haskell
test c = withContainer c $ do
  start False []
  wait ContainerRunning (-1)
  freeze
  wait ContainerFrozen (-1)
  unfreeze
  wait ContianerRunning (-1)
  stop
  wait ContainerStopped (-1)
```

I guess it would be nice to have `start`, `stop` and others to wait by default. And introduce something like `startAsync`, `stopAsync` and similar for current behaviour.
